quotes/QuoteModel.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+/*
+ Below is from: https://github.com/lukePeavey/quotable
+ {
+    "length" : 76,
+    "_id" : "VDDodhULiss2",
+    "content" : "It is through science that we prove, but through intuition that we discover.",
+    "tags" : [
+       "famous-quotes",
+       "science",
+       "technology"
+    ],
+    "authorSlug" : "henri-poincare",
+    "author" : "Henri Poincaré"
+ }
+*/
+
+struct Quote: Codable {
+    let length: Int
+    let id, content: String
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/swift-linting-formatting-workshop/blob/2f007754112214b4f5103cca96bc3418eef48e0d/quotes/QuoteModel.swift#L21"></span>Variable name should be between 3 and 40 characters long: 'id'<br />
<code>identifier_name</code> <code>QuoteModel.swift:21</code></td>
    </tr>
  </tbody>
</table>